saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -371,11 +371,25 @@ def perform_mutation(cls, _root, info, ids, **data):
             ).values_list("pk", flat=True)
         )
 
+        product_pks = list(
+            models.Product.objects.filter(variants__in=pks)
+            .distinct()
+            .values_list("pk", flat=True)
+        )
+
         response = super().perform_mutation(_root, info, ids, **data)
 
         # delete order lines for deleted variants
         order_models.OrderLine.objects.filter(pk__in=order_line_pks).delete()
 
+        # set new product default varaint if any has been removed
```

Comment:
You have been visited by the grammar police 👮  
```suggestion
        # set new product default variant if any has been removed
```